### PR TITLE
update web sdk release version number

### PR DESCRIPTION
**Pull request by Kirkkt**
Minor.

@derrickstaten - I will work on updating the `release.sh` in web sdk so that we don't have to manually update these in the future.

### CODE REVIEW

File: ingredients/web_sdk/send_sms_example.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ Here is a fully-functional web page that you can use as a template for your text
     <head>
         <meta charset="UTF-8">
         <script type="text/javascript">
-            (function(b,r,a,n,c,h,_,s,d,k){if(!b[n]||!b[n]._q){for(;s<_.length;)c(h,_[s++]);d=r.createElement(a);d.async=1;d.src="https://cdn.branch.io/branch-v1.6.1.min.js";k=r.getElementsByTagName(a)[0];k.parentNode.insertBefore(d,k);b[n]=h}})(window,document,"script","branch",function(b,r){b[r]=function(){b._q.push([r,arguments])}},{_q:[],_v:1},"init data addListener removeListener setIdentity logout track link sendSMS referrals credits creditHistory applyCode validateCode getCode redeem banner closeBanner".split(" "), 0);
+            (function(b,r,a,n,c,h,_,s,d,k){if(!b[n]||!b[n]._q){for(;s<_.length;)c(h,_[s++]);d=r.createElement(a);d.async=1;d.src="https://cdn.branch.io/branch-v1.6.7.min.js";k=r.getElementsByTagName(a)[0];k.parentNode.insertBefore(d,k);b[n]=h}})(window,document,"script","branch",function(b,r){b[r]=function(){b._q.push([r,arguments])}},{_q:[],_v:1},"init data addListener removeListener setIdentity logout track link sendSMS referrals credits creditHistory applyCode validateCode getCode redeem banner closeBanner".split(" "), 0);
```
Comment (Kirkkt) [inline ingredients/web_sdk/send_sms_example.md]: it's a long line but the only change I made is to change 1.6.1 to 1.6.7

**Comment by dmitrig01** [comment]
we have an _initialization ingredient this should include - maybe @derrickstaten you can help kirk get that going?

**Comment by derrickstaten** [comment]
@dmitrig01 that's over in [documentation](https://github.com/BranchMetrics/documentation/). How would you propose we update that? We definitely need something automated :/

**Comment by dmitrig01** [comment]
@derrickstaten this is documentation

**Comment by derrickstaten** [comment]
oh oops. yeah. we need to consolidate.

**Comment by derrickstaten** [comment]
:+1:

**Comment by dmitrig01** [comment]
oh i wanted to consolidate in this PR. can you show kirk how to consolidate?

**Pull request merged**